exams/tasks.py

```diff
@@ -118,6 +121,8 @@ def export_exam_authorizations(self):
         try:
             # upload to SFTP server
             upload.upload_tsv(tsv.name)
+        except ImproperlyConfigured as ice:
+            log.warning('export_exam_authorizations got an ImproperlyConfigured exception: %s', ice)
```

Comment:
After discussion with George [over here](https://github.com/mitodl/micromasters/pull/2447#discussion_r99610047), could you change this to a `log.exception`? Also, could add a check for the feature flag `PEARSON_EXAMS_SYNC` (e.g. `settings.FEATURES.get("PEARSON_EXAMS_SYNC", False)`) so that we can ensure this task doesn't execute and then error in environments we aren't explicitly turning this on and configuring it in?